### PR TITLE
feat(database): creation flow

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

[> Link to the JIRA ticket](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-515)

Database creation flow from start to finish

CleanShot 2022-10-06 at 13 31 36@2x

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(database): creation flow - 2022-10-06T11:31:56Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/1ee004a2-4c3e-4421-81f1-156840c63db6/applications)

 Another comment will be posted when deployments are terminated

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://z1ee004a2-z4d2a7643-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://z1ee004a2-ze649ecd5-gtw.za8ad0659.bool.sh)

**Review by RemiBonnet — COMMENTED** [review]
Well done @bdebon good job working well ! ⚡ 

I added some small comments and I noticed an UI bug when we create an application or database on the service list, we have this (at the left of the deploy button): 
Capture d’écran 2022-10-06 à 14 04 05


And, can you add an hover for this button (the button with wheel) for db and app ?
Capture d’écran 2022-10-06 à 14 03 03

File: libs/shared/ui/src/lib/components/modals/modal-confirmation/modal-confirmation.tsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ export function ModalConfirmation(props: ModalConfirmationProps) {
             required: 'Please enter a name.',
             validate: (value) => value === name || 'Please enter the right name.',
           }}
+          defaultValue=""
```
[the remark this answers is not in the document]
Comment (bdebon) [reply libs/shared/ui/src/lib/components/modals/modal-confirmation/modal-confirmation.tsx:64]: We had a js error anytime we were using the confirm box. So there were two way to resolve this, either by putting defaultValue to `''` or by setting the defaultValue to `{ name: '' }` when we do the `useForm`.
Here it was quicker to put defaultValue='' but would your prefer that I do it like this:

CleanShot 2022-10-06 at 14 18 32@2x

Both works.

**Pull request merged**